server/templates/private_dc/iitm/base.html

```diff
@@ -58,11 +56,12 @@
 
 <body id="iitm-india">
   <div id="main">
+    
+    {% if is_homepage %}
+    <nav class="navbar navbar-light navbar-expand-lg fixed-top " id="mainNav" style="padding: 0px 0px;">
+      <div class="container-fluid mr-5" style="padding: 0px;">
+    {% else %}
     <header id="main-header">
```

Comment:
I couldn't figure this out  -- why is the header element dropped from the html for the homepage?

Reply:
Homepage uses a different navbar design than other pages. We don't use the solid color bar for navbar in homepage. It's just the gradient and logos. So, we added the header element directly in homepage.html

Normal navbar with solid purple bar
![image](https://user-images.githubusercontent.com/30504842/166112496-69cbf324-b43e-4181-93e9-22419551847e.png)

Homepage navbar with gradient and logos
![image](https://user-images.githubusercontent.com/30504842/166112511-52573e1a-9eca-49c7-a672-39d29f909831.png)

Reply:
ok got it. i mistook it for the header element which is now used for the hero image. thanks!